src/BenchmarkDotNet/Jobs/JobExtensions.cs

```diff
@@ -52,7 +52,7 @@ public static class JobExtensions
         public static Job WithMinIterationTime(this Job job, TimeInterval value) => job.WithCore(j => j.Accuracy.MinIterationTime = value);
         public static Job WithMinInvokeCount(this Job job, int value) => job.WithCore(j => j.Accuracy.MinInvokeCount = value);
         public static Job WithEvaluateOverhead(this Job job, bool value) => job.WithCore(j => j.Accuracy.EvaluateOverhead = value);
-        public static Job WithRemoveOutliers(this Job job, bool value) => job.WithCore(j => j.Accuracy.RemoveOutliers = value);
```

Comment:
@AndreyAkinshin I think that we could keep `WithRemoveOutliers` for backward compatibility and implement it to keep the old behavior. Just to have fewer breaking changes.

```cs
[Obsolete("Please use the new WithOutlierMode instead")]
public static Job WithRemoveOutliers(this Job job, bool value) 
    => job.WithOutlierMode(value ? OutlierMode.All : OutlierMode.None);
```

Reply:
@adamsitnik fixed.